package.json

```diff
@@ -32,7 +32,12 @@
     "lint_fix": "eslint . --fix",
     "generate_words": "babel-node scripts/generate_words.js",
     "generate_wallets": "babel-node scripts/generate_wallets.js",
-    "get_xpub_from_seed": "babel-node scripts/get_xpub_from_seed.js"
+    "get_xpub_from_seed": "babel-node scripts/get_xpub_from_seed.js",
+    "docs_convert": "babel-node scripts/convert-docs.js",
+    "docs_lint": "npx @redocly/cli lint tmp/api-docs.json",
+    "docs_server": "npx @redocly/cli preview-docs tmp/api-docs.json",
+    "docs_clean": "rm tmp/api-docs.json",
+    "docs": "npm run docs_convert && npm run docs_lint && npm run docs_server && npm run docs_clean"
```

Comment:
The new script "converts" the api-docs and saves it to `tmp`, why not use known build folder names like `dist` or `build`?

Reply:
The idea for this `json` file is to actually be something temporary, and not incorporated into the application in any form - as it is not today. This is reinforced by having the `docs_clean` command: allowing for easy removal of the file at the end of the local validation.

Since the `tmp` folder is usually present at the development environment because of the integration tests, this seemed like a good output folder.

Should we decide to have this as a more persistent development asset, a `build` or `dist` folder could be created.

Reply:
Why don't you save it in the root project folder? It's just one html file, isn't it?

Reply:
I'd say this file still belongs to the `tmp` folder instead of `build`, `dist` or the root folder because the file is not used anywhere else outside of this script's scope.

The `[GET] /docs` endpoint, for example, does not use this generated file, and queries its `api-docs.js` file as a source of truth.

In a future moment, as an expansion of this feature, we could event use the `concurrently` dependency to monitor the server lifespan, watch for changes on the `api-docs.js` file and actually delete the generated `json` file right after the server process is closed.

What do you think about this?